cmd/nri-vsphere/nri-vsphere.go

```diff
@@ -87,6 +88,14 @@ func checkAndSanitizeConfig(config *load.Config) {
 		config.Logrus.Fatal("missing argument `pass`, please check if password has been supplied")
 	}
 
+	if config.Args.EnableVspherePerfMetrics && config.Args.PerfMetricFile ==""{
+		if runtime.GOOS == "windows" {
+			config.Args.PerfMetricFile = "C:\\Program Files\\New Relic\\newrelic-infra\\integrations.d\\vsphere-performance.metrics"
+		} else {
+			config.Args.PerfMetricFile = "/etc/newrelic-infra/integrations.d/vsphere-performance.metrics"
+		}
```

Comment:
I would not use this literal string for the paths. Please use **filepath.Join()** and **filepath.Abs()** to "build" the path to the file